rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -590,3 +590,15 @@ def snp_noise(*inputs, snpNoise=None, device=None, preserve = False):
     kwargs_pybind = {"input_image0":inputs[0], "is_output":False ,"snpNoise": snpNoise}
     snp_noise_added_image = b.SnPNoise(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
     return (snp_noise_added_image)
+
+def external_source(*inputs, source, device=None, color_format=types.RGB, random_shuffle=False,mode=types.EXTSOURCE_FNAME, max_width=2000, max_heigth=2000):
```

Comment:
Please add space before and after , 
random_shuffle=False,mode=types.EXTSOURCE_FNAME